nix/postgres/setup-script.nix

```diff
@@ -79,7 +83,10 @@ in
   name = "setup-postgres";
   runtimeInputs = with pkgs; [ config.package coreutils gnugrep gawk ];
   text = ''
-    set -euo pipefail
+    set -x
+    function psql_with_args() {
+      psql ${psqlUserArg} -v "ON_ERROR_STOP=1" "$@"
+    }
```

Comment:
Add a comment for this function. 

Also, any reason not to inline (or move to a `let` inside of `text =`) `psqlUserArg` here? Furthermore, the `ON_ERROR_STOP` seems redundant here as `psqlUserArg` already has it above.